OpenTrons/opentrons-api
# fix(protocol-designer): fix layout and menu items of SlotOverflowMenu

# Overview

This PR fixes the layout and content of SlotOverflowMenu component. It aligns text to the left of the menu button. Also, it always renders 'Add liquid' option, but disables if no labware is present in the slot. Last, it adds a Divider above 'Clear slot' option.

Closes RQA-3335

## Test Plan and Hands on Testing

- create or edit a protocol starting deck where some slots contain labware and others are empty
- verify that 'Add liquid' menu item always shows, but is disabled if no labware is present in the slot
- verify that divider line shows above 'clear slot'
- verify that all text is aligned left

## Changelog

- update `MenuButton` css
- fix logic for showing liquid menu item
- add divider

## Review requests

- see test plan

## Risk assessment

low
